starcheck/src/starcheck.pl

```diff
@@ -44,13 +42,18 @@
 use HTML::TableExtract;
 
 use Ska::AGASC;
+use Carp 'verbose';
+$SIG{ __DIE__ } = sub { Carp::confess( @_ )};
```

Comment:
With this change, if there is an error, the starcheck STDERR output will be more messy, and will be messy even if the problem is on the Perl side and not the Python side.  I think this is acceptable for our users (SOT/FOT MP and us).